huntsman/camera/pyro.py

```diff
@@ -600,7 +726,20 @@ def get_uid(self):
 
     def take_exposure(self, seconds, base_name, dark, dir_name=None, *args,
                       **kwargs):
-        
+        '''
+        Parameters
+        ----------
+        base_name (str):
+            The basename of the file.
+        dir_name (str, optional):
+            The directory in which to store the file. If None, will use
+            the images directory indicated by self.config.
```

Comment:
I like to point out that this `base_name`, `dir_name` split was only introduced to make the rsync file transfer work. Now that you've removed the need for that with the sshfs mount the directory name manipulation is just unnecessary faffing around. You should remove the code on the `Camera` side that splits up the `filename` and the code on the `CameraServer` side that replaces the `dir_name`, and have this `take_exposure` method just take a `filename` like the methods of the `Camera` classes do. As long as we make sure the `images` directories on both ends have the same path then everything is then a lot simpler.

Deleting code is even better than writing code. We should do it every chance we get!